fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/patient/PatientExportPartitionMapper.java

```diff
@@ -61,13 +74,46 @@ public PartitionPlan mapPartitions() throws Exception {
         ConfigurationAdapter adapter = ConfigurationFactory.getInstance();
         adapter.registerRequestContext(ctx.getTenantId(), ctx.getDatastoreId(), ctx.getIncomingUrl());
 
+        // Check resourceType needs to be processed
```

Comment:
smart

Reply:
@lmsurpre if you can re-review, thanks, Paul

Reply:
Great - it should be a bit better